action.yml

```diff
@@ -2,60 +2,51 @@
 
 name: Run MATLAB Tests
 description: >-
-  Run all tests in a MATLAB project and generate test artifacts
+  Run MATLAB and Simulink tests and generate artifacts
 inputs:
   source-folder:
     description: >-
-      Source code folder Location of the folder containing source code, relative
-      to the project root folder. The specified folder and its subfolders are
-      added to the top of the MATLAB search path. To generate a code coverage
-      report, MATLAB uses only the source code in the specified folder and its
-      subfolders. You can specify multiple folders using a colon-separated or a
-      semicolon-separated list.
+      Source code folder location of the folder containing source code
     required: false
     default: ""
   select-by-folder:
     description: >-
       Locations of the folders used to select test suite elements, relative to
-      the project root folder. To create a test suite, MATLAB uses only the
-      tests in the specified folders and their subfolders. You can specify
-      multiple folders using a colon-separated or semicolon-separated list.
+      the project root folder
     required: false
     default: ""
   select-by-tag:
     description: >-
-      Test tag used to select test suite elements. To create a test suite,
-      MATLAB uses only the test elements with the specified tag.
+      Test tag used to select test suite elements
     required: false
     default: ""
   test-results-pdf:
     description: >-
-      Path to write test results report in PDF format (currently not supported
-      on macOS platforms).
+      Path to write test results report in PDF format (not supported
+      on macOS)
     required: false
     default: ""
   test-results-junit:
     description: >-
-      JUnit-style test results Path to write test results report in JUnit XML
-      format.
+      JUnit-style test results path to write test results report in JUnit XML
+      format
     required: false
     default: ""
   test-results-simulink-test:
     description: >-
-      Path to export Simulink Test Manager results in MLDATX format (requires
-      Simulink Test license and is supported in MATLAB R2019a or later).
+      Path to export Simulink Test Manager results in MLDATX format
```

Comment:
For model coverage, you mention the release requirement. How about in here? Or you might leave out the parenthetic details altogether if they are not critical.

Reply:
Good catch; I'm not sure actually if we consider that critical 🤔 
@acampbel @mcafaro thoughts? 🤔

Reply:
yeah i think
it can be in the doc, not needed in the one liner description

Reply:
Sounds good. I've cut the parenthetic data from `model-coverage-cobertura` to match this one 😄